cmd/registry/cmd/export-csv_test.go

```diff
@@ -45,8 +47,8 @@ func TestExportCSV(t *testing.T) {
 	err = client.DeleteProject(ctx, &rpc.DeleteProjectRequest{
 		Name: "projects/" + projectID,
 	})
-	if err != nil {
-		t.Fatalf("Setup: Failed to delete project: %s", err)
+	if err != nil && status.Code(err) != codes.NotFound {
```

Comment:
LGTM - were tests sometimes failing because these deletes failed to complete?

Reply:
They were passing locally because the test doesn't delete the project after it finishes, so this wasn't returning "Not found" on my local runs. When I pushed it up and CI ran it against an empty database, it started to fail.

This happens to me frequently and is only possible because our tests are non-hermetic. As much as I try to stop the server, rebuild, restart it, delete the databases, etc. it's difficult to avoid these issues.